reference/gdscript.rst

```diff
@@ -808,7 +808,7 @@ There are 6 pattern types:
                 print("the name is dennis")
             {"name": "dennis", "age": var age}:
                 print("dennis is ", age, " years old.")
-            {"name", "age"}:
+            {"name", "key": "godotisawesome", "age"}:
```

Comment:
BTW, this would be catched even if `name=dennis` in the cases where `key=godotisawesome`, since none of the above matches is open-ended. (just mentioning)